### PR TITLE
remove unused messages

### PR DESCRIPTION
**Pull request by elharo**
@patflynn @loosebazooka

### CODE REVIEW

**Comment by patflynn** [comment]
LGTM

File: google-cloud-tools-plugin/resources/messages/SyncingBundle.properties [hunk 1]
```diff
@@ -1,16 +0,0 @@
-#
```
Comment (loosebazooka) [inline google-cloud-tools-plugin/resources/messages/SyncingBundle.properties]: perhaps we should remove this file completely?

Comment (elharo) [inline google-cloud-tools-plugin/resources/messages/SyncingBundle.properties]: I thought I did.

Comment (loosebazooka) [inline google-cloud-tools-plugin/resources/messages/SyncingBundle.properties]: haha, you did, I should have looked at hte file summary.

**Pull request merged**